sphinx/users/index.rst

```diff
@@ -35,6 +35,7 @@ for carrying out a variety of tasks:
     :includehidden:
 
     comlinetools/index
```

Comment:
Unrelated to this PR, the nested toctree here is confusing and inconsistent with the rest of the users documentation. I'll likely open a separate PR to turn this into a flat list.

Reply:
#391 is now merged and conflicts are resolved.